envs/env.py

```diff
@@ -368,6 +391,9 @@ def reset(self) -> GymObs:
         self.target_episode = 0
         self.effective_episode = 0
 
+        self.random_variation_step = self.set_random_variation_step()
+        self.link_variation(1.0, 100)
```

Comment:
These shouldn't be hardcoded, but default values for the beginning of the episode. Let's have some class parameters for this as well, we can name them something like `bandiwdth_start` and `latency_start` and the variations `bandwidth_var` and `latency_var`